### PR TITLE
Todo

### PR DESCRIPTION
**Pull request by degisew**
### ToDo List App
**In this PR I implemented the following features:**

-  Setup project with CRA
-  Using Components
-  Using Props and state
-  Using Hooks with function-based components
-  Using React Router 
-  Using React Icons

### CODE REVIEW

File: src/classBased/components/TodoContainer.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  state = {
+    todos: [],
+  };
+
+  // fetch data from Json Placeholder API
+  // componentDidMount() {
+  //   fetch("https://jsonplaceholder.typicode.com/todos?_limit=10")
+  //     .then((response) => response.json())
+  //     .then((data) => this.setState({ todos: data }));
+  // }
```
Comment (Tommymarvel) [inline src/classBased/components/TodoContainer.js:18]: Kindly remove the dead code, to make your code clean :+1:

File: src/functionBased/pages/About.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+/* eslint-disable */
+import React from "react";
+
+const About = () => (
+  //   const { url, path } = useRouteMatch();
```
Comment (Tommymarvel) [inline src/functionBased/pages/About.js:5]: Kindly remove the dead code.

**Review by Tommymarvel — CHANGES_REQUESTED** [review]
Hi @degisew   

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
 - [x] Correct GitH  flow practice :heavy_check_mark:
 - [X] Linters are passing :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by saikatgpae** [comment]
> ### ToDo List App
> **In this PR I implemented the following features:**
> 
> * Setup project with CRA
> * Using Components
> * Using Props and state
> * Using Hooks with function-based components
> * Using React Router
> * Using React Icons

APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
Hey @degisew  😊 😃
Great Job 😄 👏 You've done it. Your project is complete!

Highlights 🚀 ✔️ ✔️
The repo looks professional ✔️
Git/GitHub flow best practices have been followed ✔️
You have a descriptive PR ✔️
Professional README 💯
Meaningful commit messages ✔️
No linter errors ✔️
The app works as expected ✔️
There is nothing else to say other than... it's time to merge it 👍🏻 ✅

Congratulations!!! 🎉 🥳 👏 🙌 🎊
Cheers and Happy coding!👏👏👏

**Pull request merged**